packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -19,16 +19,12 @@ import type {
     Hint,
     ChangeHandler,
     DeviceType,
+    ImageUploader,
 } from "@khanacademy/perseus";
 
 const {InfoTip, InlineIcon} = components;
 const {iconCircleArrowDown, iconCircleArrowUp, iconPlus, iconTrash} = icons;
 
-type ImageUploader = (
-    file: string,
```

Comment:
This was a duplicate type, but it also typed `file` incorrectly (it should be `File`).